pkg/mcs/scheduling/server/grpc_service.go

```diff
@@ -191,6 +185,48 @@ func (s *Service) StoreHeartbeat(ctx context.Context, request *schedulingpb.Stor
 	return &schedulingpb.StoreHeartbeatResponse{Header: &schedulingpb.ResponseHeader{ClusterId: s.clusterID}}, nil
 }
 
+// SplitRegions split regions by the given split keys
+func (s *Service) SplitRegions(ctx context.Context, request *schedulingpb.SplitRegionsRequest) (*schedulingpb.SplitRegionsResponse, error) {
```

Comment:
Do we need to add some tests about forward requests?

Reply:
Yes, I have added a TODO in the PR description. Will do it later.